src/components/FormInput.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Input, Icon } from "./SharedStyles";
+
+const FormInput = ({ icon, altText, placeholder }) => {
```

Comment:
1. Nitpick: Can we rename `altText` to `iconAltText` so we are more explicit about what the prop is used for? 
2. Can we add additional props for `Input`?
   - `id`: used for `id` and `name` attributes to avoid using the same id and name in multiple input components. 
   - `label`: used for `aria-label` so we are ensuring that our text input is accessibility-friendly.